### PR TITLE
enh(Resource-status/Details): Format command line to make it better readable 

### PR DESCRIPTION
**Pull request by bdauria**
## Description

This splits the command from its arguments in the details panel so that it's easier to read for the User:

Capture d’écran 2021-05-04 à 21 53 56

**Fixes** # (issue)

## Type of change

- [x] Patch fixing an issue (non-breaking change)

## Target serie

- [ ] 20.04.x
- [ ] 20.10.x
- [ ] 21.04.x
- [x] 21.10.x (master)

<h2> How this pull request can be tested ? </h2>

Please describe the **procedure** to verify that the goal of the PR is matched. Provide clear instructions so that it can be **correctly tested**.

Any **relevant details** of the configuration to perform the test should be added.

## Checklist

- [ ] I have followed the **coding style guidelines** provided by Centreon
- [ ] I have commented my code, especially new **classes**, **functions** or any **legacy code** modified. (***docblock***)
- [ ] I have commented my code, especially **hard-to-understand areas** of the PR.
- [ ] I have made corresponding changes to the **documentation**.
- [ ] I have **rebased** my development branch on the base branch (master, maintenance).

### CODE REVIEW

File: www/front_src/src/Resources/Details/tabs/Details/CommandLine/utils/index.tsx [hunk 1]
```diff
@@ -0,0 +1,58 @@
+import { filter, equals, isNil, find, pipe, head } from 'ramda';
+import commandParser from 'string-argv';
+
+const isShortArgument = (argument: string): boolean => {
+  return argument.startsWith('-') && argument.length === 2 && argument !== '--';
```
Comment (Thebarda) [inline www/front_src/src/Resources/Details/tabs/Details/CommandLine/utils/index.tsx:5]:
```suggestion
  return startsWith('-', argument) && equals(length(argument), 2) && not(equals('--', argument));
```

**Comment by Thebarda** [comment]
Screenshot 2021-04-14 at 13 27 17

Here is the result with a pipe

**Comment by bdauria** [comment]
> Screenshot 2021-05-05 at 17 15 50
> 
> Here is the result with a pipe
> @bdauria I think we should split the command for each pipes before parsing the command but keep display it.

I don't really find it disturbing to have the pipe on a newline personnally

**Comment by bdauria** [comment]
> > Screenshot 2021-05-05 at 17 15 50
> > Here is the result with a pipe
> > @bdauria I think we should split the command for each pipes before parsing the command but keep display it.
> 
> I don't really find it disturbing to have the pipe on a newline personnally

Hmm maybe a little, i'll split up the command

**Comment by Thebarda** [comment]
> > Screenshot 2021-05-05 at 17 15 50
> > Here is the result with a pipe
> > @bdauria I think we should split the command for each pipes before parsing the command but keep display it.
> 
> I don't really find it disturbing to have the pipe on a newline personnally

That's not what I meant ^^. The pipe on a new line is 👌 . The `grep` should be displayed as a command (black rather than gray). That's why I recommend to split these commands ^^

**Comment by bdauria** [comment]
@Thebarda 

Updated version: 

Capture d’écran 2021-05-10 à 11 21 20

File: www/front_src/src/Resources/Details/tabs/Details/CommandLine/index.tsx [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+const useStyles = makeStyles((theme) => ({
+  argument: {
+    marginRight: theme.spacing(0.5),
+  },
+  argumentWithValue: {
+    display: 'flex',
+    marginLeft: theme.spacing(1),
+  },
+  command: {
+    fontWeight: 'bold',
```
Comment (Thebarda) [inline www/front_src/src/Resources/Details/tabs/Details/CommandLine/index.tsx:18]:
```suggestion
    fontWeight: theme.typography.fontWeightBold,
```

**Review by Thebarda — APPROVED** [review]

**Review by jdelpierre — APPROVED** [review]

**Pull request merged**